certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json

```diff
@@ -829,14 +874,14 @@
                                                 "properties": {
                                                     "value": {
                                                         "type": "string",
-                                                        "description": "Admin Contact State",
+                                                        "description": "State or province of the administrative contact",
                                                         "title": "Admin Contact State"

```

Comment:
@RamboV please look through the changes and make sure that strings are in complete form and everything is clear.